### DISCUSSION TITLE
optimize Help-Command

### DISCUSSION THREAD

**Discussion by zerobot-issues** [EinfachRuwen/ZeroBot#16] [General]
Add MarkDown-Files into the GitHub-Repository, so you can make a list of commands in groups. For example: Group A = General, Group B = Fun, Group C = Mod (Issue created by Discord-user `byZero#4840`)

**Comment by therealmistericraft** [comment]
I recommend using the GitHub wiki instead.

**Comment by EinfachRuwen** [comment]
thats right, my head is hurting now cause I was putting it on my desk to hard :)

**Comment by EinfachRuwen** [comment]
hey :) @therealmistericraft i optimized the _help command, how about that?

**Reply by therealmistericraft** [reply]
Its OK, I would prefer if the commands are seen directly in discord. Opening the browser every time is annoying. But I can really understand that you do not want to have multiple full embeds.

**Reply by EinfachRuwen** [reply]
Hmm, that's one thing I should change. It's annoying myself.

**Comment by EinfachRuwen** [comment]
@therealmistericraft ?
